packages/sync/src/modules/Users.php

```diff
@@ -635,12 +646,12 @@ public function estimate_full_sync_actions( $config ) {
 	/**
 	 * Retrieve the WHERE SQL clause based on the module config.
 	 *
-	 * @access private
+	 * @access public
 	 *
 	 * @param array $config Full sync configuration for this sync module.
 	 * @return string WHERE SQL clause, or `null` if no comments are specified in the module config.
 	 */
-	private function get_where_sql( $config ) {
+	public function get_where_sql( $config ) {
```

Comment:
Should we update the method access in the WooCommerce module for consistency?

Reply:
No.